src/createButton.ts

```diff
@@ -12,11 +14,26 @@ export interface ButtonFactory extends ComposeFactory<Button, ButtonOptions> { }
 
 const createButton: ButtonFactory = createWidget
 	.mixin({
-		mixin: createFormFieldMixin
+		mixin: createFormFieldMixin,
+		aspectAdvice: {
+			before: {
+				getNodeAttributes(this: Button, overrides: VNodeProperties = {}) {
```

Comment:
Did we want to wait until #40 is landed and refactor this?  It should be heck of a lot easier.